### PR TITLE
Specify tam name which program should work with

### PR DESCRIPTION
**Pull request by DamianW93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ GENERAL|GENERAL_LOGOUT|Logout|Wyloguj|...
 Check out sample translation sheet [here](https://docs.google.com/spreadsheets/d/1HZwNTyo2XRkADjNLvXlMMlZ_buzCNYxhjIzvwL6mid8/)
 
 2. Click on **Share** button, and open **Advanced** panel
-3. Invite **<your account>@sheetstranslator.iam.gserviceaccount.com** with **view** permissions
+3. Invite **<your account>translator@sheetstranslator.iam.gserviceaccount.com** with **view** permissions
```
Comment (m-chojnacki) [inline README.md]: I think it should be either:
`Invite **<your account>@sheetstranslator.iam.gserviceaccount.com** with **view** permissions`
or
`Invite **translator@sheetstranslator.iam.gserviceaccount.com** with **view** permissions`

not both 🤔

File: SheetsTranslator/main.swift [hunk 2]
```diff
@@ -26,7 +26,13 @@ private let outputArgument =
                           required: false,
                           usage: "Output translation path (or current directory if not provided)")
 
-CommandLineTool.shared.parser.add(arguments: sheetArgument, credentialsArgument, outputArgument)
+private let tabNameArgument =
+CommandStringArgument(shortFlag: "t",
+                      longFlag: "tab",
+                      required: false,
+                      usage: "Sheet tab name(or first one if not provided)")
```
Comment (m-chojnacki) [inline SheetsTranslator/main.swift]:
```suggestion
                      usage: "Sheet tab name (or first one if not provided)")
```

File: SheetsTranslator/main.swift [hunk 3]
```diff
@@ -26,7 +26,13 @@ private let outputArgument =
                           required: false,
                           usage: "Output translation path (or current directory if not provided)")
 
-CommandLineTool.shared.parser.add(arguments: sheetArgument, credentialsArgument, outputArgument)
+private let tabNameArgument =
+CommandStringArgument(shortFlag: "t",
```
Comment (m-chojnacki) [inline SheetsTranslator/main.swift]: tab ->

File: README.md (hunk 1 above)
Comment (DamianW93) [reply README.md]: Oh, I see my mistake

**Review by m-chojnacki — APPROVED** [review]
sztos

**Pull request merged**